addons/fortify/XEH_postInit.sqf

```diff
@@ -24,7 +24,7 @@ GVAR(objectRotationZ) = 0;
 
 [QGVAR(sideBudgetHint), {
     params ["_side"];
-    if (_side isEqualTo side group ACE_player && {GVAR(settingHint) isEqualTo 2 || {GVAR(settingHint) isEqualTo 1 && {"ACE_Fortify" in (items ACE_player)}}}) then {
+    if (_side isEqualTo side group ACE_player && {GVAR(settingHint) isEqualTo 2 || {GVAR(settingHint) isEqualTo 1 && {"ACE_Fortify" in (ACE_player call ACEFUNC(common,uniqueItems))}}}) then {
```

Comment:
Could use the uniquePlayerItems thing with extra caching

Reply:
still wip: https://github.com/acemod/ACE3/pull/6594

Reply:
I keep forgetting that things are not merged. Same with the "Interaction to open weapon holders and dead body inventory"